tensorboard/plugins/core/core_plugin_test.py

```diff
@@ -53,6 +67,19 @@ def testRoutesProvided(self):
     self.assertIsInstance(routes['/data/logdir'], collections.Callable)
     self.assertIsInstance(routes['/data/runs'], collections.Callable)
 
+  def testFlag(self):
```

Comment:
If you're adding this test anyway, could you add a test for the inspect=False case as well?  E.g. ensuring that if fails if neither logdir or db is passed, but succeeds if either one is present.